academic/demo/res_company_demo.xml

```diff
@@ -18,6 +18,7 @@
         <field name="report_header">ASOCIACIÓN PARA LA PROMOCIÓN DEPORTIVA, EDUCATIVA Y SOCIAL</field>
         <!-- <field name="partner_id" ref="main_partner"/> -->
         <field name="currency_id" ref="base.ARS"/>
+        <!-- <field name="fiscalyear_last_day">31</field> -->
```

Comment:
Apply this line and the lines below:

Why the comment lines?

It's not use, better delete them